public/platform/WebScreenInfo.h

```diff
@@ -77,6 +77,8 @@ struct WebScreenInfo {
     // It is the opposite of the physical rotation.
     uint16_t orientationAngle;
 
```

Comment:
I applied your comment and committed the changes.
(1) add descriptions for the deviceRadius
(2) add deviceRadius to the initialization lists